index.js

```diff
@@ -96,6 +107,50 @@ function parse(str, options) {
  */
 
 function serialize(name, val, options) {
+  if( typeof name === 'object') {
+    var cookies = name;
+    var serializeOptions = val;
+
+    var cookieNames = Object.keys(cookies);
+    if(0 === cookieNames.length) {
+      return '';
```

Comment:
Perhaps return `undefined` for this case? Not too clear, but just feel like setting the `Set-Cookie` header to the result would make more sense if `undefined` was returned, since that would just not send the header instead of send an empty header.